content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,328 @@
+using `GET` method when the request includes a `version` header of value `v1` or `v2`.
+Any requests not matched with the policy will be denied by default.
+
+That's all you need to enable access control for a workload, compared to the `v1alpha1`
+RBAC policy, you would need to configure three resources to achieve the same result:
+
+{{< text yaml >}}
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ClusterRbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON_WITH_INCLUSION'
+  inclusion:
+    services: ["httpbin.foo.svc.cluster.local"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: httpbin
+  namespace: foo
+spec:
+  rules:
+  - services: ["httpbin.foo.svc.cluster.local"]
+    methods: ["GET"]
+    constraints:
+    - key: request.headers[version]
+      values: ["v1", "v2"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRoleBinding
+metadata:
+  name: httpbin
+  namespace: foo
+spec:
+  subjects:
+  - user: "cluster.local/ns/default/sa/sleep"
+  roleRef:
+    kind: ServiceRole
+    name: "httpbin"
+{{< /text >}}
+
+### Workload selector
+
+One major change in the `v1beta1` authorization policy is it now uses workload selector
+to specify where to apply the policy. This is the same workload selector used in
+the `Gateway`, `Sidecar` and `EnvoyFilter` configurations.
+
+The workload selector makes it clear that the policy is applied and enforced on workloads
+instead of services. If a policy applies to a workload that is used by multiple different
+services, the same policy will affect the traffic to all the different services.
+
+You can simply leave the `selector` empty to apply the policy to all workloads
+in a namespace. The following policy applies to all workloads in the namespace `bar`:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: policy
+ namespace: bar
+spec:
+ rules:
+ # omitted
+{{< /text >}}
+
+### Root namespace
+
+A policy in the root namespace applies to all workloads in the mesh in every namespaces.
+The root namespace is configurable in the [`MeshConfig`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig)
+and has the default value of `istio-system`.
+
+For example, you installed Istio in `istio-system` namespace and deployed workloads in
+`default` and `bookinfo` namespace. The root namespace is configured to `istio-config`.
+The following policy will apply to workloads in every namespace including `default`,
+`bookinfo` and also the `istio-system`.
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: policy
+ namespace: istio-config
+spec:
+ rules:
+ # omitted
+{{< /text >}}
+
+### Ingress/Egress Gateway support
+
+The `v1beta1` authorization policy can also be applied on ingress/egress
+gateway to enforce access control on traffic entering/leaving the mesh, you just
+need to change the `selector` to make the policy to select the ingress/egress workload.
```

Comment:
when gateway is used within selector, it would be good to mention if the authz policy applies to all the gateway resources that refers to the same selector scope?

Reply:
The authz policy only applies to workloads in the same namespace as the policy, unless the authz policy is applied in the root namespace. Updated.